src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,96 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_RESIZE = 2;
+    private Node<K, V>[] table;
+    private int currentLoad;
+    private int size;
 
-    @Override
-    public void put(K key, V value) {
+    static class Node<K, V> {
```

Comment:
you need to make class Node private and better to move it to the end of the class